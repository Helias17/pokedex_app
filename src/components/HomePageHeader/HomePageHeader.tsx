import { HeaderMenu } from "../HeaderMenu";
import css from "./homePageHeader.module.scss";

export const HomePageHeader = () => {
  return (
    <header className={css.pageHeader}>
      <HeaderMenu />
      <h1 className={css.pageHeader__title}>Pokedex</h1>
    </header>
  );
};
