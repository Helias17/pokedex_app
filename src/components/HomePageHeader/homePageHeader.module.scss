@import "../../assets/styles/variables.scss";

.pageHeader {
  position: relative;
  height: 150px;
  overflow: hidden;
  margin-bottom: -45px;

  &__title {
    position: absolute;
    left: $indent-smart;
    top: 65px;
    font-size: 24px;
    font-weight: bold;
    color: #35353a;
  }
}
