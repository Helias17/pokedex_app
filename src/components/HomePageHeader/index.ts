import { HomePageHeader } from "./HomePageHeader";

export { HomePageHeader };
