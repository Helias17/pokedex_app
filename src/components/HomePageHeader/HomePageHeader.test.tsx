import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import { HomePageHeader } from "../HomePageHeader";

describe("Homepage header", () => {
  test("Homepage header to match snapshop", () => {
    const { asFragment } = render(<HomePageHeader />);

    expect(asFragment()).toMatchSnapshot();
  });
});
