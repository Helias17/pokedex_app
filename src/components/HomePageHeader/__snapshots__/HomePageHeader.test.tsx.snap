// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage header Homepage header to match snapshop 1`] = `
<DocumentFragment>
  <header
    class="pageHeader"
  >
    <div
      class="menu"
    />
    <h1
      class="pageHeader__title"
    >
      Pokedex
    </h1>
  </header>
</DocumentFragment>
`;
